app/lib/import_export.rb

```diff
@@ -141,7 +141,7 @@ def xlsx_export(config, patients, outer_batch_size, inner_batch_size)
 
   # Gets data for this batch of patients that may not have already been present in the export config (such as specific symptoms).
   def get_field_data(config)
-    data = config[:data].deep_dup
+    data = {}.merge!(config[:data])
```

Comment:
out of curiosity, how is this different performance/memory-wise than calling `deep_dup`?

Reply:
I think my benchmark on this showed it was 2.25x faster

Reply:
Lol I was way off

```
Warming up --------------------------------------
              merge!   194.542k i/100ms
            deep_dup    31.152k i/100ms
Calculating -------------------------------------
              merge!      2.259M (±22.6%) i/s -     10.700M in   5.044433s
            deep_dup    254.886k (±24.0%) i/s -      1.215M in   5.011937s

Comparison:
              merge!:  2258869.3 i/s
            deep_dup:   254886.3 i/s - 8.86x  (± 0.00) slower
```